### PR TITLE
piece of diagnostic migrate

### PR DESCRIPTION
**Pull request by AndyJado**
r? @davidtwco

### CODE REVIEW

File: compiler/rustc_borrowck/src/borrow_set.rs [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#![deny(rustc::untranslatable_diagnostic)]
+#![deny(rustc::diagnostic_outside_of_impl)]
```
Comment (Nilstrieb) [inline compiler/rustc_borrowck/src/borrow_set.rs:2]: Could you deny it in the whole `rustc_borrowck` crate in `lib.rs` instead, if that works already? If it doesn't you can still remove at least some of the denies in submodules. `#![deny]` attribute applies transitively for all submodules as well.

Comment (AndyJado) [reply compiler/rustc_borrowck/src/borrow_set.rs:2]: there would be still 200 more errors messages if I deny it in lib..

I just trimmed submodules, did I get it right?

File: compiler/rustc_borrowck/src/diagnostics/mod.rs [hunk 2]
```diff
@@ -597,9 +597,32 @@ impl UseSpans<'_> {
 
     // Add a span label to the use of the captured variable, if it exists.
     // only adds label to the `path_span`
```
Comment (davidtwco) [inline compiler/rustc_borrowck/src/diagnostics/mod.rs:599]:
```suggestion
    /// Add a span label to the use of the captured variable, if it exists.
    /// only adds label to the `path_span`
```

**Review by davidtwco — APPROVED** [review]
LGTM, one nit then r=me

**Pull request merged**